cni/network/invoker_cns.go

```diff
@@ -175,9 +175,16 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 				return IPAMAddResult{}, err
 			}
 		default:
+			// Assign default index
+			if ifIndex == -1 {
+				// add a default Interface
+				// would put within configureDefaultAddResult() but we have a dependency on numInterfacesWithDefaultRoutes
+				addResult.interfaceInfo = append(addResult.interfaceInfo, network.InterfaceInfo{NICType: cns.InfraNIC})
```

Comment:
shouldn't this come from CNS response?

Reply:
Because we only want to create a single infra nic. Until we make another PR where I can use maps to check if the infra nic exists within addResult I am leveraging `ifIndex` to be both an index and a flag.

Reply:
but CNS response will only have 1 infra nic too no?

Reply:
It will have 1 infra nic, but multiple podIPInfo associated when it is dualstack scenario.